### DISCUSSION TITLE
Let's Encrypt one click SSL

### DISCUSSION THREAD

**Discussion by manakill** [btcpayserver/btcpayserver#2438] [Technical Support] [answered]
Would be awesome to have the Let's Encrypt one click SSL Certificate installer, and also work on Subdomain.  This way users wouldn't get the "Security Warning" about the connection not being secure. I could probably do it manually but then I'd also have to renew it manually, when I know it can be automated.

**Comment by pavlenex** [comment, marked as answer]
For Docker Deployments Let's Encrypt is already there and renews automatically.

**Reply by manakill** [reply]
Ok, so in a MyNode implementation, I see the cert was signed by MyNode, and they are using Docker - Looks like this is an issue with them then, yes?

**Reply by Kukks** [reply]
Yes, this is their issue.
